### ISSUE TITLE
Entitity name without release name

### ISSUE THREAD

**Issue by Tri0L** [nixys/nxs-universal-chart#9]
Hello!

Is it possible to create entities without release name?

For example I have one app with name my-app.
After command `helm install my-app nixys/universal-chart -f my-values.yaml` I want to see Deployment, Service, Pod, Ingress with name `my-app`, not with `my-app-my-app`.

my-values.yaml:
```
deployments:
  my-app:
    replicas: 1

services:
  my-app:
    type: "ClusterIP"
    ports:
      - name: http
        protocol: TCP
        port: 8080
```

Result:
```
# Source: universal-chart/templates/svc.yml
kind: Service
apiVersion: v1
metadata:
  name: my-app-my-app
  namespace: "default"
  labels:
    app.kubernetes.io/name: my-app
    app.kubernetes.io/instance: my-app
    helm.sh/chart: universal-chart-2.0.0
    app.kubernetes.io/managed-by: Helm
  annotations:
spec:
  type: ClusterIP
  ports:
  - name: http
    protocol: TCP
    port: 8080
  selector:
    app.kubernetes.io/name: my-app
    app.kubernetes.io/instance: my-app
---
# Source: universal-chart/templates/deployment.yml
apiVersion: apps/v1
kind: Deployment
metadata:
  name: my-app-my-app
  namespace: default
  labels:
    app.kubernetes.io/name: my-app
    app.kubernetes.io/instance: my-app
    helm.sh/chart: universal-chart-2.0.0
    app.kubernetes.io/managed-by: Helm
  annotations:    
spec:
  replicas: 1
  selector:
    matchLabels:
      app.kubernetes.io/name: my-app
      app.kubernetes.io/instance: my-app
  template:
    metadata:
      labels:
        app.kubernetes.io/name: my-app
        app.kubernetes.io/instance: my-app
      annotations:
    spec:
      affinity:
        nodeAffinity:
          {}
        podAffinity:
          preferredDuringSchedulingIgnoredDuringExecution:
          - podAffinityTerm:
              labelSelector:
                matchLabels:
                  app.kubernetes.io/name: my-app
                  app.kubernetes.io/instance: my-app
              namespaces:
              - "default"
              topologyKey: kubernetes.io/hostname
            weight: 1
        podAntiAffinity:
          preferredDuringSchedulingIgnoredDuringExecution:
          - podAffinityTerm:
              labelSelector:
                matchLabels:
                  app.kubernetes.io/name: my-app
                  app.kubernetes.io/instance: my-app
              namespaces:
              - "default"
              topologyKey: kubernetes.io/hostname
            weight: 1
      
      containers:
      volumes:      
        []

```

**Comment by randreev1321** [comment]
Hello.

To solve your issue you can use `releasePrefix` from [common parameters](https://github.com/nixys/nxs-universal-chart#common-parameters) with "-" value.

**Comment by kzamulin** [comment]
Hello,

I found another problem with theese prefixies. 
Setting the `releasePrefix` parameter to `-` you won't have prefix, however you won't have a prefix in the name of a secret, which is created when using `secretEnvs` or `secretEnvsString` parameter either. Secret is then created with just name `secret-envs`, without any prefixies at all, which makes it unusable in the same cluster namespace (e.g. you deploy two charts and they overwrite secrets of each other).

And I found no workaround to this problem, rather then use `app` as deployment name. Thus, if you want to have your deployment to be named in the cluster "my-deploy", you won't achive that, you will have "my-deploy-app", or anything else as suffix instead of "-app".
